common/lib/mandrill_client/client.py

```diff
@@ -63,14 +64,23 @@ def send_mail(self, template_name, user_email, context, attachments=[], subject=
         if subject:
             message.update({'subject': subject})
 
+        if receiver_emails:
+            message.update({'to': receiver_emails})
+
+        if sender_email:
+            message.update({'headers': {'Reply-To': sender_email}})
+
+        if images:
+            message.update({'images': images})
```

Comment:
just a suggestion, it would be worth breaking it down to further functions.

Reply:
yes right i have done the change